disruption_py/workflow.py

```diff
@@ -121,26 +125,39 @@ def get_shots_data(
             repeat(retrieval_settings),
             shotlist_list,
         )
+        num_success = 0
 
-        for shot_id, shot_data in pool.imap(_execute_retrieval, args):
+        for shot_id, shot_data in tqdm(
+            pool.imap(_execute_retrieval, args), total=len(shotlist_list), leave=False
+        ):
             if shot_data is None:
                 logger.warning(
-                    "Not outputting data for shot %s due, data is None.", shot_id
+                    shot_log_msg(
+                        shot_id, "Not outputting data for shot, data is None."
+                    ),
                 )
             else:
+                num_success += 1
                 output_setting.output_shot(
                     OutputSettingParams(
                         shot_id=shot_id,
                         result=shot_data,
                         database=database,
                         tokamak=tokamak,
-                        logger=logger,
                     )
                 )
+        total = len(shotlist_list)
+        percent_success = round(num_success / total * 100, 2)
```

Comment:
was there a reason for rounding? I removed the rounding in the other instance when I modified it.